static/js/taxpayer_creation.js

```diff
@@ -4,12 +4,64 @@ var left, opacity, scale;
 var animating; 
 
 
-$(".next").click(function(){
+// Checks if first part of form is complete
+function check_first_fieldset_complete(){
+	var token = true
+	$(".mandatory>input:text, .mandatory>input[type='number'], .mandatory>select, .mandatory>input[type='email']").each(function(index){
+		if ($(this).val() == "") {
+			token = false;
+		}
+		else if($(this).get(0).checkValidity() == false){
+			token = false;
+		}
+	});
+	return token
+}
+
+// Click on First "next" button
+$("#first-next").click(function(){
```

Comment:
Apparently this function follows the same logic as `$("#arg-bank-next").click` only changing the selector maybe we can think in a function that receives a selector and execute this logic.

Reply:
its the same logic but they call to diferent functions, one checks the completion of the first field set, and the other one checks the completion of Bank argentina fieldset